lib/pharos/phases/label_node.rb

```diff
@@ -59,7 +66,7 @@ def find_node
       end
 
       def kube_nodes
-        kube_client.api('v1').resource('nodes')
+        @kube_nodes ||= kube_client.api('v1').resource('nodes')
```

Comment:
I assume there aren't going to be new nodes during the phase, so memoizing here.